apps/emqx/src/emqx_banned.erl

```diff
@@ -126,7 +130,7 @@ parse(Params) ->
             By = maps:get(<<"by">>, Params, <<"mgmt_api">>),
             Reason = maps:get(<<"reason">>, Params, <<"">>),
             At = maps:get(<<"at">>, Params, erlang:system_time(second)),
-            Until = maps:get(<<"until">>, Params, At + 5 * 60),
+            Until = maps:get(<<"until">>, Params, At + ?EXPIRATION_TIME),
```

Comment:
Q: can't the default value when `<<"until">>` is absent really be `infinity`?

It's a bit of a hack, but since atoms are always greater than numbers:

```elixir
iex(10)> :infinity > 100000000000000000
true
```

:joy_cat: 

Perhaps because this value has to be returned in some API, and must be a number?

Reply:
I also think using `permanent` or `infinite` will be better, but they are not friendly for the external caller.